src/other/Table.js

```diff
@@ -34,6 +34,7 @@
 
     Table.prototype.publish("fixedSize", false, "boolean", "Fix Size to Min Width/Height");
 
+    Table.prototype.publish("updateOnRowSelection", true, "boolean", "Call update on row select/deselect", null, { tags: ["Private"] });
```

Comment:
Can you change this to:
"quickUpdateOnClick" and default to false?

Reply:
(I am open to better name suggestions - but would like it to be "true" to override the default behavior.

Reply:
Done